[2024-05-20T20:12:02.881+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-20T20:12:02.947+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.call_api manual__2024-05-20T20:12:00.491298+00:00 [queued]>
[2024-05-20T20:12:02.964+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.call_api manual__2024-05-20T20:12:00.491298+00:00 [queued]>
[2024-05-20T20:12:02.964+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-20T20:12:02.997+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): call_api> on 2024-05-20 20:12:00.491298+00:00
[2024-05-20T20:12:03.006+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=167) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-20T20:12:03.009+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'call_api', 'manual__2024-05-20T20:12:00.491298+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpe91i36yk']
[2024-05-20T20:12:03.010+0000] {standard_task_runner.py:63} INFO - Started process 168 to run task
[2024-05-20T20:12:03.010+0000] {standard_task_runner.py:91} INFO - Job 11: Subtask call_api
[2024-05-20T20:12:03.093+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_dag.call_api manual__2024-05-20T20:12:00.491298+00:00 [running]> on host 1c53b53fc77d
[2024-05-20T20:12:03.317+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='call_api' AIRFLOW_CTX_EXECUTION_DATE='2024-05-20T20:12:00.491298+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-20T20:12:00.491298+00:00'
[2024-05-20T20:12:03.319+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-20T20:12:03.414+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-20T20:12:03.415+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-20T20:12:03.433+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=call_api, run_id=manual__2024-05-20T20:12:00.491298+00:00, execution_date=20240520T201200, start_date=20240520T201202, end_date=20240520T201203
[2024-05-20T20:12:03.479+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-20T20:12:03.520+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-20T20:12:03.522+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
